util/pivnet_meta.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+slug=$1
+version=$(pivnet releases -p $1 --format json | jq -r '.[0].version')
```

Comment:
Can you validate a version was found? If a version can't be found it might mean the tile slug was invalid

Reply:
Also, can you add a line at the beginning to ensure `pivnet` is on the classpath, and if it isn't, print the github repo?